### PR TITLE
Qt4 Merge in Master #2

### PR DESCRIPTION
**Pull request by tst-cjeffries**
Merge in master.

### CODE REVIEW

**Comment by tst-dsugar** [comment]
I know this is ugly with autogen code...it will be good to see how this matches up with master post-merge.

**Review by tst-dsugar — APPROVED** [review]

**Review by tst-basilhuffman — APPROVED** [review]

File: docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+|Google logo|
```
Comment (tst-lsavoie) [inline docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst:1]: All of the file accessor plugin stuff was pulled out of master. Maybe this branch is not completely up to date?

Comment (tst-cjeffries) [reply docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst:1]: When was that done?

Comment (tst-cjeffries) [reply docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst:1]: Getting master into this branch was a mess.

Comment (tst-lsavoie) [reply docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst:1]: Looks like it happened on Sept 4: https://github.com/google/earthenterprise/commit/ec83a3f1d7e68e6fb9b7419e50f73cc1c9cbc514

**Comment by tst-lsavoie** [comment]
@tst-dsugar is there something specific you want me to review on this? I'm not going to be able to go through all of the changes.

File: docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst (hunk 1 above)
Comment (tst-cjeffries) [reply docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst:1]: I can try to make another merge from master.

Comment (tst-lsavoie) [reply docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst:1]: Hypothetically you'll eventually merge master into the QT4 branch again at some time in the future, so you could possibly wait until then.

Comment (tst-cjeffries) [reply docs/geedocs/docsrc/answer/developers/creatingFileAccessorPlugin.rst:1]: Right. The next step was to create a wip toward master. Changes that are unintentional should be fairly obvious, and can be fixed at that point. For now I am going to check if I can pull anything else from master, and then we will fix things as part of the next PR.

**Pull request merged**